ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java

```diff
@@ -51,6 +53,7 @@ public void clickActiveTab() {
     }
 
     public void clickInActiveTab() {
+    	waiter.until(ExpectedConditions.elementToBeClickable(TAB_INACTIVE));
```

Comment:
Collect the identation here. Make sure its on the same line.

Reply:
where's the indentation to correct here @sherrif10 ?
you want `clickOn(TAB_INACTIVE);` and `waiter.until(ExpectedConditions.elementToBeClickable(TAB_INACTIVE));` on the same line?